eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -9,30 +9,8 @@ parameters:
   PublishCodeCoverage: false
   TestBrowser: true
   TestSamples: true
+  TestMinMax: false
```

Comment:
Should this be true by default?

Reply:
actually all the packages don't have the test segregation. It's only one package - keyvault-secrets for now

Reply:
Eventually yes, but currently the majority of packages do not support min/max testing, so it is opt-in for now.

Reply:
so should we add like TestMinMax: false in other tests.yml is what you are suggesting?

Reply:
Ideally I want to move these matrix elements to the main matrix in archetype-tests.yml like this one - https://github.com/Azure/azure-sdk-for-js/blob/98d2e2f4fa60b42e62868075643375ab9254bead/eng/pipelines/templates/jobs/archetype-sdk-integration.yml#L13-L27

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/8919

Reply:
My opinion is `TestMinMax` should be opt-in (default to `false`) until most packages support it.  Then the default should be switched to `true`, it should be removed from packages which do support it, and `TestMinMax: false` should be added to the packages which don't support it.